src/components/dragBoxLayer.ts

```diff
@@ -289,7 +289,7 @@ export module Components {
 
     // Sets resizable classes. Overridden by subclasses that only resize in one dimension.
     protected _setResizableClasses(canResize: boolean) {
```

Comment:
Possibly outside scope of this pull request, but this function really doesn't need a parameter if `canResize === this._resizable`